### DISCUSSION TITLE
Demos for version 6.8.4

### DISCUSSION THREAD

**Discussion by geekashu** [nolimits4web/swiper#4922] [Q&A]
Is it possible to get v6.8.4 demos and examples as we have for v.7.0.3? 

I am not able to upgrade from v6.8.4 to v7.0.3 as I created my application from CRA and CRA doesn't seems to support ESM only packages.

BTW, I tried creating a fresh CRA app and import this package but it didn't work (Unable to resolve.... error). I am not sure what needs to be done to make 7.0.3 work with CRA.

**Comment by hirbod** [comment]
Yes. We need an archived / versioned documentation

**Comment by 144mdgross** [comment]
Yes archived / versioned docs would be incredibly helpful. I need to add swiper to nextJS 10 which only supports CJS modules, and finding the right information for v6.x is challenging.

**Comment by Arhell** [comment]
https://github.com/nolimits4web/swiper/tree/Swiper6
